speech/livecaption/livecaption.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Command livecaption pipes the stdin audio data to
+// Google Speech API and outputs the transcript.
+//
+// As an example, gst-launch can be used to capture the mic input:
+//
+//    $ gst-launch-1.0 -v pulsesrc ! audioconvert ! audioresample ! audio/x-raw,channels=1,rate=16000 ! filesink location=/dev/stdout | livecaption
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"golang.org/x/net/context"
+	"google.golang.org/api/option"
+	"google.golang.org/api/transport"
+	speech "google.golang.org/genproto/googleapis/cloud/speech/v1beta1"
+)
+
+func main() {
+	ctx := context.Background()
+	conn, err := transport.DialGRPC(ctx,
+		option.WithEndpoint("speech.googleapis.com:443"),
+		option.WithScopes("https://www.googleapis.com/auth/cloud-platform"),
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer conn.Close()
+
+	// [START speech_streaming_mic_recognize]
+	stream, err := speech.NewSpeechClient(conn).StreamingRecognize(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+	// send the initial configuration message.
```

Comment:
sentence casing, please (and in the two comments below)

`// Send the initial configuration message.`